cmd/repl/repl.go

```diff
@@ -97,15 +115,15 @@ func main() {
 		}
 	}
 
-	for {
-		fmt.Print("sqlflow> ")
-		slct := readStmt()
-		fmt.Println("")
-
+	isTerminal := !flagPassed("execute", "e", "file", "f") && terminal.IsTerminal(syscall.Stdin)
+	sqlRun := func(stmt string) {
+		if !isTerminal {
+			fmt.Println("sqlflow>", stmt)
```

Comment:
I am not sure why we need to print `sqlflow>` in a non-terminal mode such as `go run repl.go -e "select 1;"`.

Reply:
In my own experience, user may use redirection to save information in the training process,  such as `go run repl.go -f 'train_and_evaluate.sql' **> train.log 2>&1**`, `sqlflow> ` can be used as an indicator for different output blocks.